src/standardTypes.js

```diff
@@ -1,10 +1,15 @@
+const List = ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`;
+
 export default {
   Int: () => 't.Number',
   Float: () => 't.Number',
   String: () => 't.String',
   Boolean: () => 't.Boolean',
   Date: () => 't.String',
   DateTime: () => 't.String',
-  List: ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`,
-  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`
+  List,
+  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`,
+  Set: List,
```

Comment:
mmm I'm not sure.. what about the ability to write/read literals: we'd need a fromJSON/toJSON convention in place:
```js
JSON.stringify(new Set(['foo'])) // => {}

// httpapi would need to
SetType.fromJSON(json)
// and in some way
JSON.stringify([...setInstance])
```